docs/source/tutorials/map_styling.rst

```diff
@@ -0,0 +1,68 @@
+.. _map_styling:
+
+Cartography and Map Styling
+=============================
+
+Copyright note
+--------------
+
+Since basemap data is taken from external sources, it is advised to cite and credit the data provider accordingly.
+
+Some data providers request or at least recommend to include a corrsponding note along with the map.
+Depending on the policy, this may either be only the include the data source and licese or terms of use.
+
+The corresponding information for the data providers included in cartopy is listed in the :ref:`data_copyright_table`:
+
+.. todo::
+
+    Turn typical licenses into http://docutils.sourceforge.net/docs/ref/rst/directives.html#replacement-text
+    
+.. todo::
+
+    maybe even add the typical annotation text as an string property to each function / class
+
+.. _data_copyright_table:
+
+.. csv-table:: List of Data Providers and Form of Citation
+   :file: ../_static/copyright_license.csv
+   :delim: ;
+   :header-rows: 1
+   :widths: 10, 10, 10, 10, 10, 10
+
+.. |copy| unicode:: 0xA9 .. copyright sign
+.. |TM| unicode:: U+2122
+   .. with trademark sign
+.. |---| unicode:: U+02014 .. em dash
+   :trim:
+
+
+
+
+Here is an :ref:`example <data_copyright_plot>` showing such annotation for Natural Earth data:
+
+
+.. _data_copyright_plot:
+
+.. plot::
+    :include-source:
+
+    import cartopy.crs as ccrs
+    import cartopy.feature as cfeature
+    import matplotlib.pyplot as plt
+    from mpl_toolkits.axes_grid.anchored_artists import AnchoredText
```

Comment:
I see there's a version of this class as `matplotlib.offsetbox.AnchoredText` in matplotlib-proper. I'm not sure when it was added, but it would be better to use that one if possible.